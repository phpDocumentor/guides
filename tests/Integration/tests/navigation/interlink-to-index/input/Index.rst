==============
Document Title
==============

Lorem Ipsum Dolor.

See the :doc:`TYPO3 documentation <t3coreapi:Index>`
