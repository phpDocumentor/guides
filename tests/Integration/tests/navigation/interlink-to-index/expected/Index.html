<!-- content start -->
    <div class="section" id="document-title">
    <h1>Document Title</h1>

            <p>Lorem Ipsum Dolor.</p>
            <p>See the <a href="https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/Index.html">TYPO3 documentation</a></p>
    </div>

<!-- content end -->
